regression-test/suites/query_p0/aggregate/push_filter_through_agg.groovy

```diff
@@ -17,6 +17,7 @@
 
 suite("push_filter_through_agg") {
     sql """DROP TABLE IF EXISTS t_push_filter_through_agg"""
+    sql """DROP VIEW IF EXISTS view_i"""
```

Comment:
"create view XXX if exists" is better

Reply:
create view XXX if exists may not handle the situation that the structure of the table changed by somebody